2015/proposals/agarwal-rajat.md

```diff
@@ -0,0 +1,92 @@
+# Title
+
+Add taxonomic name resolution to the EcoData Retriever to facilitate data science approaches to ecology
+
+## Abstract
+
+The EcoData Retriever is a Python based tool for automatically downloading, cleaning up, and restructuring ecological data. It does the data munging so that the users can focus on using the data.
+As in ecological and evolutionary data, the names of species are constantly being redefined - it is difficult to combine datasets. By automating reconciliation of different species names as part of the process of accessing the data in the first place it will become much easier to combine diverse datasets and in new and interesting ways.
+
+## Technical Details
+
+The project would consists of the following components:
+
+1. Querying the taxonomy name resolution service:
+APIs of the name resolution services have to be used to fetch the possible names using Python. The name of the species would be the input to the APIs and their possible names would be returned. This will be a separate module for the purpose of reusability.      
```

Comment:
Are you planning on creating a new module or using https://github.com/sckott/pytaxize/. Being specific will help make for a more compelling application, even if you end up needing to adjust the plan later.